README.md

```diff
@@ -60,13 +60,13 @@ Other documentation in the [docs](docs) folder.
 
 ## Contributing
 
-1. Branch off of `main`, name your branch related to the feature you're
-   implementing, prefix with your initials (e.g. `mm/feature-name`)
+1. Branch off of `main`, name your branch related to the feature you're implementing, prefix with your initials (e.g. `mm/feature-name`)
 2. Do your thing
-3. Ensure unit tests pass locally with `yarn test`, and integration tests with `yarn cypress run`
+3. Ensure CI passes by running `make ci` or `./ci.sh`
```

Comment:
Oh, I didn't know this - nice!

Reply:
Off PR but maybe we could have a `make validate` which doesn't run cypress = for the speed of it 🤷 - it's hard to get this stuff right

Reply:
I think `make ci` or `./ci.sh` should be good for someone new to the team/project to get used to running everything.

Once experienced you don't have to follow these instructions rigidly, e.g just run `yarn test` for the unit tests.

The pre-push hooks also catch quite a bit too.